[CI] GenerativeForkIT test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10709 / oraclelinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/jturp2wblik76)
- [elasticsearch-pull-request #93688 / part-3](https://gradle-enterprise.elastic.co/s/qxt3wazozaffi)
- [elasticsearch-pull-request #93612 / part-3](https://gradle-enterprise.elastic.co/s/mz527h3jpg7so)
- [elasticsearch-pull-request #93565 / part-3](https://gradle-enterprise.elastic.co/s/4dvvhmucy2h5k)
- [elasticsearch-pull-request #93370 / part-3](https://gradle-enterprise.elastic.co/s/hiieytya5agjs)
- [elasticsearch-pull-request #93329 / part-3](https://gradle-enterprise.elastic.co/s/5ngwc6q7sidrq)
- [elasticsearch-pull-request #93300 / part-3](https://gradle-enterprise.elastic.co/s/przvwhx574uio)
- [elasticsearch-pull-request #93265 / part-3](https://gradle-enterprise.elastic.co/s/r4pauqz6widtc)
- [elasticsearch-pull-request #93248 / part-3](https://gradle-enterprise.elastic.co/s/jmwjgtekzvdyg)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT" -Dtests.method="test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}" -Dtests.seed=659EAB2C2370661B -Dtests.locale=doi -Dtests.timezone=Etc/GMT-1 -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bcsv-spec%3Ainlinestats.DocsMultiAggsMultiGroupings%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Data mismatch:
row 4 column 2:a list containing
row 4 column 2:0: expected "58644.0" but was "50836.666666666664"
row 4 column 3:a list containing
row 4 column 3:0: expected "2" but was "3"
Actual:
emp_no:integer | salary:integer | avg_salary:double  | count:long | languages:integer | tenure:integer
10001          | 57305          | 51130.5            | 2          | 2                 | 39
10002          | 56371          | 40180.0            | 3          | 5                 | 39
10004          | 36174          | 30749.0            | 2          | 5                 | 38
10005          | 63528          | 63528.0            | 1          | 1                 | 36
10007          | 74572          | 50836.666666666664 | 3          | 4                 | 36

Expected:
emp_no:integer | salary:integer | avg_salary:double | count:long | languages:integer | tenure:integer
10001          | 57305          | 51130.5           | 2          | 2                 | 39
10002         
[truncated]
```

**Issue Reasons:**
- [main] 9 failures in test test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings} (2.2% fail rate in 407 executions)
- [main] 8 failures in step part-3 (3.4% fail rate in 232 executions)
- [main] 8 failures in pipeline elasticsearch-pull-request (3.6% fail rate in 225 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/docs/reference/query-languages/esql/_snippets/commands/examples/inlinestats.csv-spec/multi-agg-multi-grouping.md
+++ b/docs/reference/query-languages/esql/_snippets/commands/examples/inlinestats.csv-spec/multi-agg-multi-grouping.md
@@ -4,7 +4,7 @@
 FROM employees
 | WHERE still_hired
 | KEEP emp_no, languages, salary, hire_date
-| EVAL tenure = DATE_DIFF("year", hire_date, now())
+| EVAL tenure = DATE_DIFF("year", hire_date, "2025-09-18T00:00:00")
 | DROP hire_date
 | INLINE STATS avg_salary = AVG(salary), count = count(*) BY languages, tenure
 ```

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -591,24 +591,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
   method: test {csv-spec:fork.ForkWithStats}
   issue: https://github.com/elastic/elasticsearch/issues/135116
-- class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135117
-- class: org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135119
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlSpecIT
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135118
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecIT
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135118
-- class: org.elasticsearch.xpack.esql.CsvTests
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135120
-- class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
-  method: test {csv-spec:inlinestats.DocsMultiAggsMultiGroupings}
-  issue: https://github.com/elastic/elasticsearch/issues/135121
 - class: org.elasticsearch.xpack.esql.heap_attack.HeapAttackIT
   method: testLookupExplosionBigString
   issue: https://github.com/elastic/elasticsearch/issues/135122

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/inlinestats.csv-spec
@@ -91,7 +91,7 @@ required_capability: inline_stats
 FROM employees
 | WHERE still_hired
 | KEEP emp_no, languages, salary, hire_date
-| EVAL tenure = DATE_DIFF("year", hire_date, now())
+| EVAL tenure = DATE_DIFF("year", hire_date, "2025-09-18T00:00:00")
 | DROP hire_date
 | INLINE STATS avg_salary = AVG(salary), count = count(*) BY languages, tenure
 // end::multi-agg-multi-grouping[]

